src/test/java/org/opentripplanner/updater/vehicle_positions/VehiclePositionParsingTest.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.updater.vehicle_positions;
+
+import java.net.URI;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class VehiclePositionParsingTest {
+
+    @Test
+    public void parseFirstPositionsFeed() {
+        var vehiclePositionSource = getVehiclePositionSource("king-county-metro-1.pb");
```

Comment:
This test is for parsing protobuf files.

I have another test which focusses on the logic of the matcher: https://github.com/ibi-group/OpenTripPlanner/blob/cdb3e047a372c17414cf94ceab988340b6921eff/src/test/java/org/opentripplanner/updater/vehicle_positions/VehiclePositionsMatcherTest.java#L135-L143

Do you expect more?

Reply:
Maybe we could have tests that test that the updates are parsed correctly into the internal model without issues. I think it's fine to test the parsing of a file this way but anything more than that probably should be done in code instead of in files.

Reply:
The test I linked to does test the mapping into the internal model and builds the entities in code.

As per @t2gran's request I'm trying not too add too many integration tests. In this case here I would have to add a GTFS file so that all the entities can be map.

Reply:
Also, the Seattle feed, which these updates are for, is 15MB in size which I don't really want to commit to the OTP repo.

Reply:
Ok this might a bit more complicated than the alerts which are not automatically mapped into OTP objects but instead we just keep a reference.

Reply:
I could keep an in-between state that is after parsing but before being matched so I can test it. It doesn't seem worth it in this case.